modules/utxo-lib/src/bitgo/Unspent.ts

```diff
@@ -110,6 +124,66 @@ export function addToTransactionBuilder<TNumber extends number | bigint>(
   txb.addInput(txid, vout, sequence, script, value);
 }
 
+export function addToPsbt(
+  psbt: UtxoPsbt<UtxoTransaction<bigint>>,
+  u: WalletUnspent<bigint>,
+  rootSigner: WalletUnspentSigner<RootWalletKeys>,
+  network: Network
+): void {
+  const { txid, vout, script, value } = toPrevOutput(u, network);
+  const { walletKeys, signerIndex, cosignerIndex } = rootSigner.deriveForChainAndIndex(u.chain, u.index);
+  const scriptType = scriptTypeForChain(u.chain);
+  psbt.addInput({
+    hash: txid,
+    index: vout,
+  });
+  const inputIndex = psbt.inputCount - 1;
```

Comment:
Why is this `- 1` and not `+ 1`? Since we're adding an input to a tx, I'd expect that input would increase the input count and go at the end of the index.

Reply:
We _just_ added it, so the index (0-base) of the one we just added is the count (1-base) minus one.